### PR TITLE
Reduce patch set

### PR DESCRIPTION
**Pull request by iwahbe**
Automatic upgrades are being blocked by patch conflicts. This PR reduces the patch set without reducing the quality of our documentation.

### CODE REVIEW

File: scripts/upstream.sh [hunk 1]
```diff
@@ -168,15 +168,15 @@ end_rebase() {
...
+  if [ -d "${rebase_merge_dir}" ] || [ -d "${rebase_apply_dir}" ]; then
     echo "rebase still in progress in './upstream'. Please resolve the rebase in"
     echo "'./upstream' and then run 'make \"$1\"' again."
     exit 1
   fi
 
   rm patches/*.patch
   cd upstream
-  git format-patch local -o ../patches --no-prefix --zero-commit --no-signature --no-stat
+  git format-patch local -o ../patches --zero-commit --no-signature --no-stat
```
Comment (mjeffryes) [inline scripts/upstream.sh:179]: is there a reason to remove --no-prefix?

File: patches/0001-Docs-changes.patch [hunk 2]
```diff
@@ -0,0 +1,166 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Ian Wahbe <ian@wahbe.com>
+Date: Tue, 24 Oct 2023 15:23:27 -0700
+Subject: [PATCH 1/3] Docs changes
+
```
Comment (mjeffryes) [inline patches/0001-Docs-changes.patch:5]: nit: I don't think need to have these headers?

**Review by mjeffryes — APPROVED** [review]

File: scripts/upstream.sh (hunk 1 above)
Comment (iwahbe) [reply scripts/upstream.sh:179]: Yes. https://github.com/pulumi/ci-mgmt/pull/698/commits/88e3be43b4ec35e537a748333f301d31b4520950 has details.

File: patches/0001-Docs-changes.patch (hunk 2 above)
Comment (iwahbe) [reply patches/0001-Docs-changes.patch:5]: This is the prefix that `git format-patch` produces. I'm not aware of a way to remove it that doesn't break `git am`.

Comment (mjeffryes) [reply patches/0001-Docs-changes.patch:5]: hmmm... maybe we shouldn't be using `git am` then; oh well, can deal with that at another time.

**Pull request merged**